src/Jackett.Server/Services/ServerService.cs

```diff
@@ -145,14 +144,16 @@ public void Initalize()
                         //Hard minimum of 5.8
                         //5.4 throws a SIGABRT, looks related to this which was fixed in 5.8 https://bugzilla.xamarin.com/show_bug.cgi?id=60625
 
-                        logger.Error("Your mono version is too old. Please update to the latest version from http://www.mono-project.com/download/");
+                        logger.Error(
+                            "Your mono version is too old. Please update to the latest version from http://www.mono-project.com/download/");
```

Comment:
Mono

Reply:
Not sure what you meant by this one, so I left it. Made all the other changes as suggested.

Reply:
Mono is a name and should be with capital letter. It's not caused by this PR, doesn't matter.